walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,23 @@
 
 package com.google.sps;
 
+import java.lang.*;
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    if(events.equals(Collections.emptySet())) return Arrays.asList(TimeRange.WHOLE_DAY);
```

Comment:
It seems a convention (at least for C++ and a glimpse on some java code as well) that always use {} to embrace the body of if/else/for, etc., even if there is only one statement inside it. That way, the scope is more clear, and can better avoid accidental dangle if/elseif, etc.

Reply:
To check whether a Collection type is empty, use "events.isEmpty()".